vips/vip-491/bsctestnet.ts

```diff
@@ -0,0 +1,179 @@
+export const USDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+const vip491 = () => {
+  const meta = {
+    version: "v2",
+    title: "[sepolia] New sUSDe and USDe market in the Core pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      // Add sUSDe Market
+      // oracle config
+      {
+        target: sepolia.REDSTONE_ORACLE,
+        signature: "setDirectPrice(address,uint256)",
+        params: [USDe, parseUnits("1", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.CHAINLINK_ORACLE,
+        signature: "setTokenConfig((address,address,uint256))",
+        params: [[USDe, CHAINLINK_USDe_FEED, STALE_PERIOD_26H]],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [USDe, [sepolia.REDSTONE_ORACLE, sepolia.CHAINLINK_ORACLE, sepolia.CHAINLINK_ORACLE], [true, true, true]],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Market configurations
+      {
+        target: VsUSDe_CORE,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "faucet(uint256)",
+        params: [sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, sUSDe_INITIAL_SUPPLY],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_CORE,
+            parseUnits("0.72", 18), // CF
+            parseUnits("0.75", 18), // LT
+            sUSDe_INITIAL_SUPPLY, // initial supply
+            sepolia.VTREASURY,
+            parseUnits("20000000", 18), // supply cap
+            parseUnits("0", 18), // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: VsUSDe_CORE,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.05", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
```

Comment:
I would also configure the conversions, with and incentive of 3e14, as we are doing for every market on Ethereum

Reply:
7d6269b7684040d0709fc890bc68def29265e10e